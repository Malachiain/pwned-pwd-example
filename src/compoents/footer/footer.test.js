import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './index'
describe('The Footer ',()=>{
    it('It renders the footer correctly',()=>{
        const wrapper = shallow(<Footer/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})