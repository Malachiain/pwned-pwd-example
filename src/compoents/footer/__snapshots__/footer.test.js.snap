// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer  It renders the footer correctly 1`] = `
<styled.div>
  <styled.div />
  <styled.div>
    <styled.div>
      <styled.p>
        This website uses a tool built by 
        <Component
          content="Troy Hunt"
          href="https://troyhunt.com"
        />
        . It callse the pwned passwords v2 API which references a data set of 500 million passwords that have been disclosed in data breaches. Troy collects the breaches and makes them avalible at 
        <Component
          content="have I been pwned"
          href="https://haveibeenpwned.com/"
        />
         Go check out if you have been in a data breach
      </styled.p>
    </styled.div>
    <styled.div>
      <styled.label>
        Useful resources 
      </styled.label>
      <ul>
        <li>
          <Component
            content="https://haveibeenpwned.com/"
            href="https://haveibeenpwned.com/"
          />
        </li>
        <li>
          <Component
            content="https://troyhunt.com"
            href="https://troyhunt.com"
          />
        </li>
      </ul>
      <styled.label>
        Source code of this project 
      </styled.label>
      <ul>
        <li>
          <Component
            content="https://github.com/Malachiain/pwned-pwd-example"
            href="https://github.com/Malachiain/pwned-pwd-example"
          />
        </li>
      </ul>
    </styled.div>
  </styled.div>
</styled.div>
`;
